### DISCUSSION TITLE
It's been a long long time since I have applied for the github sponsors

### DISCUSSION THREAD

**Discussion by moinulmoin** [community/community#116694] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How long does it take to get approval for GitHub sponsors? Thank you.

**Comment by DHayes10** [comment]
Hi @moinulmoin, 

We appreciate you reaching out regarding your profile status. We have experienced higher than normal volumes and apologize for the delay. Our team can't seem to track down your Sponsors profile. Could you share your handle? 

Thanks, 
GitHub Sponsors

**Comment by moinulmoin** [comment]
which handle? github? it's `moinulmoin`

**Comment by DHayes10** [comment]
Hi @moinulmoin, 

Do you mind sharing your Sponsor's profile handle? We are unable to locate your account under `moinulmoin`. 

Thanks,
GitHub Sponsors

**Reply by moinulmoin** [reply]
I can't see Sponser's profile. All I have a waitlist page,[https://github.com/sponsors/moinulmoin/waitlist](https://github.com/sponsors/moinulmoin/waitlist) , I guess the Sponser's Profile would be [https://github.com/sponsors/moinulmoin](https://github.com/sponsors/moinulmoin)
![CleanShot 2024-04-04 at 04 35 33@2x](https://github.com/community/community/assets/63459189/537e8113-0734-4c4f-94c2-362bc45f2f53)

**Reply by DHayes10** [reply]
Hi @moinulmoin, 

It appears you listed your Country/Region of residence as Bangladesh. Currently, that is listed as an unsupported region. We do encourage frequently checking our Supported Regions [List](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#supported-regions-for-github-sponsors). You can also try using a fiscal host for your project until your region becomes available. Here are the [steps](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/using-a-fiscal-host-to-receive-github-sponsors-payouts) on how to set up a fiscal host. I hope this helps!

Thanks, 
GitHub Sponsors

**Comment by umer4447** [comment]
It usually comes after few hours, 
I faced the same issue once
